src/server/saved_objects/client/lib/create_id_query.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Finds a document by either its v5 or v6 format
+ *
+ * @param type The documents type
+ * @param id The documents id or legacy id
+**/
+export function createIdQuery({ type, id }) {
```

Comment:
This should be resolved in https://github.com/elastic/kibana/pull/12648/commits/90cc6e9fd84eaf6b4e6c0dbf137388248a06c252